lib/options/ui.js

```diff
@@ -0,0 +1,42 @@
+import { themeSelect, replaceTextInput, form, userInput, userList } from './elements';
+import { updateTheme, updateReplaceText } from './updateUi';
+import renderList from './renderList';
+
+export default function optionsUi(data) {
+  const theme = data.theme || 'cats';
+  const replaceText = !!data.replaceText;
+  let users = data.users || [];
+
+  themeSelect.value = theme;
+  replaceTextInput.checked = replaceText;
+  renderList(users);
+
+  themeSelect.addEventListener('change', updateTheme);
+  replaceTextInput.addEventListener('change', updateReplaceText);
+
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    const user = userInput.value.trim();
+
+    if (user && !~users.indexOf(user)) {
+      users.push(user);
+      chrome.storage.local.set({ users });
+      renderList(users);
+      userInput.value = '';
+    }
+  });
+
+  // handle removals
+  userList.addEventListener('click', (e) => {
+    const t = e.target;
+
+    if (t.className.match(/user-list-remove/)) {
```

Comment:
You could pull this RegEx out of the listener to avoid it being created every time. Very nitpicky though ;)